aspnetcore/fundamentals/dependency-injection.md

```diff
@@ -211,6 +211,28 @@ What if you find that your classes tend to have way too many dependencies being
 
 With regards to data access specifically, you can inject the `DbContext` into your controllers (assuming you've added EF to the services container in `ConfigureServices`). Some developers prefer to use a repository interface to the database rather than injecting the `DbContext` directly. Using an interface to encapsulate the data access logic in one place can minimize how many places you will have to change when your database changes.
 
+### Disposing of services
+
+The container will call `Dispose` for `IDisposable` types it creates. However, if you add an instance to the container yourself, it will not be disposed.
+
+Example:
+
+```csharp
+// Assume Service1, Service2, and Service3 each implement IDisposable
```

Comment:
It just seemed like clutter. I can add if you want, but it doesn't impact the sample any (except to show what implementing IDisposable looks like, but I think customers who care about this behavior will already be familiar with that). Your call.